### DISCUSSION TITLE
About LLM module

### DISCUSSION THREAD

**Discussion by eros1006** [DedicatedSoftwareDev/Lending-Bot#10] [Polls]
Anyone who have experience in this topic, feel free to answer your opinion and experience

**Comment by websparkledev** [comment]
Whisper API?

**Reply by websparkledev** [reply]
Whisper is an automatic speech recognition (ASR) system trained on 680,000 hours of multilingual and multitask supervised data collected from the web.

**Reply by eros1006** [reply]
Good answer

**Comment by eros1006** [comment]
Great answers
